Doc/library/zipfile.rst

```diff
@@ -385,10 +385,10 @@ ZipFile Objects
 
 .. method:: ZipFile.writestr(zinfo_or_arcname, data[, compress_type])
 
-   Write the string *data* to the archive; *zinfo_or_arcname* is either the file
-   name it will be given in the archive, or a :class:`ZipInfo` instance.  If it's
-   an instance, at least the filename, date, and time must be given.  If it's a
-   name, the date and time is set to the current date and time.
+   `ZipFile.write()` writes a file into archive. `ZipFile.writestr()` writes a bytes data or a string.
```

Comment:
Oh, I didn't mean that you should include my words verbatim. `ZipFile.write()` is documented in the description of `ZipFile.write()`.

The existing documentation is mostly good. The only problem is that the term "string" is used in wider meaning that includes byte strings, but this is not clear. Just document what actually does this method.